content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available all Team Pro and Enterprise customers to preview."
+meta_image: meta.png
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News","Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `pulumi-policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks.
+
+<!--more-->
+
+You can run policies against any Pulumi stack, independent of the stack's language. This means you can have policies written in TypeScript and run them against a Python stack. Policies are grouped in Policy Packs and run against a stack during both `pulumi preview` and `pulumi up`. Each resource in a stack is run through the Policy Pack prior to creation, meaning you can block the creation of out-of-compliance resources.
+
+## Why
+
+Using Pulumi, developers and operators are empowered to self provision their infrastructure and move quickly. One our core goals is to allow you to codify best practices and share them with your team. We often hear from organization leaders that they're concerned about how to make sure their team doesn't accidentally go off the rails. CrossGuard provides guardrails your leadership needs to allow your team to continue to operate self-sufficiently.
+
+## Key Features
+
+The key features available during the CrossGuard preview are:
+
+* TypeScript `pulumi-policy` SDK to express custom policies
+* Run Policy Packs locally using the `--policy-pack` flag
+* [AWSGuard library](https://github.com/pulumi/pulumi-awsguard) - codifies best practices for AWS
+* Enforce Policy Packs across your organization or particular stacks in the Pulumi Console
+* View policy errors and Policy Packs for an update in the Pulumi Console
+
+## Pulumi CrossGuard for Everyone
+
+Pulumi CrossGuard has something to offer everyone in your organization.
+
+### For Organization Administrators
+
+You want your organization to be able to move quickly and deliver business value. To do so, your team needs to be able to self-service their infrastructure. Pulumi CrossGuard gives you the peace of mind that you need to allow your team to work independently. Codify business and security policies to ensure your team meets the requirements you set forth.
+
+### For Security
```

Comment:
"Security" isn't a type of "Everyone". Perhaps
- For Administrators
- For Security Engineers
- For Developers and Operators